### PR TITLE
[rpc] nit: fix typo for entry description string

### PR DESCRIPTION
**Pull request by mryandao**
#12240

### CODE REVIEW

**Review by promag — COMMENTED** [review]
ACK

**Review by fanquake — APPROVED** [review]
utACK 18dee67

File: src/rpc/blockchain.cpp [hunk 1]
```diff
@@ -373,7 +373,7 @@ std::string EntryDescriptionString()
            "        \"base\" : n,         (numeric) transaction fee in " + CURRENCY_UNIT + "\n"
            "        \"modified\" : n,     (numeric) transaction fee with fee deltas used for mining priority in " + CURRENCY_UNIT + "\n"
            "        \"ancestor\" : n,     (numeric) modified fees (see above) of in-mempool ancestors (including this one) in " + CURRENCY_UNIT + "\n"
-           "        \"descendent\" : n,   (numeric) number of in-mempool ancestor transactions (including this one) in " + CURRENCY_UNIT + "\n"
+           "        \"descendent\" : n,   (numeric) number of in-mempool descendent transactions (including this one) in " + CURRENCY_UNIT + "\n"
```
Comment (MarcoFalke) [inline src/rpc/blockchain.cpp]: `GetModFeesWithDescendants` has nothing to do with the number of transactions, it returns a fee.

**Comment by jonasschnelli** [comment]
utACK f7c414daa980efc3d7aa5255a706e9c18b680694

**Comment by fanquake** [comment]
re-ACK f7c414d

**Comment by laanwj** [comment]
utACK, the field is indeed called "descendant" not "descendent"
f7c414daa980efc3d7aa5255a706e9c18b680694

**Pull request merged**